downstream/assemblies/platform/assembly-installing-controller-operator-external-db.adoc

```diff
@@ -8,48 +8,49 @@ ifdef::context[:parent-context: {context}]
 
 
 [id="installing-contr-operator-external"]
-= Installing {ControllerName} on {OCP} with an external database and routed web traffic
+= Installing {ControllerName} on {OCP} with an external database
 
 
 :context: installing-contr-operator-external
 
 
 [role="_abstract"]
-You can use these instructions to install the {HubName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
+You can use these instructions to install the {ControllerName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
 
 // mirrors AWX operator flow
 
 == Prerequisites
 
 * You have installed the catalog containing the {PlatformName} operators
 
-.Installing the {HubName} operator
+== Installing the {ControllerName} operator
 
 . Log in to {OCP}.
 . Navigate to *Operators* -> *Operator Hub*.
-. Locate the {PlatformName} catalog and click on it.
-. Click *Install*
```

Comment:
It might be useful here to say that upon successful installation of Platform Operator, a message appears with a "view Operator" button. Click this button to view the resources provided by the Platform Operator (Hub and Controller deployment and management)